roles/addons/powerman/templates/powerman.conf.j2

```diff
@@ -4,7 +4,7 @@
 
 include "/etc/powerman/ipmipower.dev"
 
-{% if icebergs_system == true %}
+{%- if icebergs_system == true %}
```

Comment:
This `-` is not needed.

Reply:
I disagree, if I remove this - and the one in line 17 I have a line created:
```
 --- before: /etc/powerman/powerman.conf                                                                                                                                                  +++ after: /root/.ansible/tmp/ansible-local-1925480kd2s2ita/tmp7xxqpibp/powerman.conf.j2                                                                                                 @@ -2,6 +2,8 @@                                                                                                                                                                           ## Ansible managed file, do not edit

 include "/etc/powerman/ipmipower.dev"
+
+
```